### PR TITLE
Data engineer technical exercise

### PR DESCRIPTION
**Pull request by tiffanyjachja**
Purpose
As per request. This is the data engineering interview assignment. 

Changes
README shared.

### CODE REVIEW

File: Data-Engineering-CYA/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Choose Your Own Adventure
+Welcome to the adventure into the Data team at Vox Media. Our goal is to become a world-class data engineering team leading the way in developing production-ready and state-of-the-art data science solutions. We do this by providing Vox Media Stakeholders with democratized and integrated data solutions that help them gain clarity, insights, and scale with first-party data. 
+## How To Prepare
+For the Data Engineer role, you will participate in a 45-technical panel interview with the Revenue Data team. Choose any ONE of the following prompts and come prepared to present and discuss.
```
Comment (techsassy) [inline Data-Engineering-CYA/README.md:4]: I'm guessing "45-technical" should be "45-minute technical"

Comment (techsassy) [reply Data-Engineering-CYA/README.md:4]: Maybe also include expectations for presenting and discussing? "come prepared to present and discuss in whichever way you prefer. For example, you can share outlined notes, walkthrough a visual presentation, or engage in detailed conversation." <--- those might not be the right words, but outlining for people is always helpful.

File: Data-Engineering-CYA/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* What is ETL?
+* What happens after a data pipeline is available?
+
+**Institute of Mathematica:**
+* * What the benefits of using MapReduce functions
+* * Explain the Pub/Sub model 
+
+**The Museum of Visual Arts:**
+* * Software engineering background or experience
+* * Do you have experience or an interest in Javascript or Data visualization?
```
Comment (techsassy) [inline Data-Engineering-CYA/README.md:31]: In all of the above, can we add question marks on all the questions? And turn the phrases into complete sentences. Making it shiny ✨

File: Data-Engineering-CYA/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+### Prompt Two: Is it the end of Data Warehousing?
+The data warehouse architecture has been in the IT industry for at least three decades now. Read the Lakehouse paper and come ready to discuss the following:
```
Comment (techsassy) [inline Data-Engineering-CYA/README.md:34]: Hyper link the Lakehouse paper to "Read the Lakehouse Paper" here.

File: Data-Engineering-CYA/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+* What are some of the considerations Data Engineers should have when architecting data platforms and pipeline processes for consumption by data stakeholders? (We will ask about reducing costs and optimizing query computation)
+* What are some other methods and designs for achieving Lakehouse goals?
+* How will you measure the impact and performance of data management?
+
+[Lakehouse: A New Generation of Open Platforms that Unify Data Warehousing and Advanced Analytics](http://cidrdb.org/cidr2021/papers/cidr2021_paper17.pdf)
+
+### Prompt Three: Project Mania- A Trip Down Memory Lane
+We’d love to see any relevant past work you have completed! Use this time to determine what to demo. Come ready to share your role in the design and implementation of the data solution. 
+
+Feel free to share any slides, links or documentation to the work ahead of time by sending an email to rp.dataproduct@voxmedia.com with your full name in the subject line.
```
Comment (techsassy) [inline Data-Engineering-CYA/README.md:45]: Does this email alias exist?

File: README.md [hunk 5]
```diff
@@ -1,61 +1,61 @@
...
 
-Additionally, pretend we work together:
+* **One** hiring exercise from the list given to you by your interview coordinator
+* **One** demo of code that you previously worked on.
 
-* What does your teammate need to know about it?
-* Did you structure it in a way that is understandable to others?
-* Is it sufficiently documented?
-* Given more time, what would you do next to improve the project that you didn't have time for here?
+Either path you choose in your own adventure is up to you as all options are weighted equally.
```
Comment (techsassy) [inline README.md:33]: Simplify to just "All options are weighted equally!"

File: README.md [hunk 6]
```diff
@@ -1,61 +1,61 @@
...
+Here is an outline of questions or topics to expect during the panel evaluation:
+* What was the use case and scope of the project? What was implemented?
+* What was your process for reading through the exercise? 
+* Why did you choose this approach? And what considerations were relevant in making this decision.
+* What errors and challenges did you run into?
+* How did you use other software programming or data skills to do this work?
+* Any future work and considerations you’d like to highlight.
 
+## Sharing and submitting
+Feel free to share any slides, links or documentation to the work ahead of time by sending an email to the hiring manager or respective interviewer with your full name in the subject line.
```
Comment (techsassy) [inline README.md:47]: "sending an email to **your recruiting contact, the** hiring manager or respective interviewer with your full name in the subject line."

**Review by techsassy — CHANGES_REQUESTED** [review]
Thank you for updating this so thoughtfully!

File: Data-Engineering-CYA/README.md (hunk 4 above)
Comment (tiffanyjachja) [reply Data-Engineering-CYA/README.md:45]: yes

**Review by techsassy — APPROVED** [review]

**Comment by techsassy** [comment]
🎉 👍

**Pull request merged**